metrics/testdata/prometheus_metrics

```diff
@@ -152,6 +152,9 @@ container_memory_mapped_file{container_env_foo_env="prod",container_label_foo_la
 # HELP container_memory_max_usage_bytes Maximum memory usage recorded in bytes
 # TYPE container_memory_max_usage_bytes gauge
 container_memory_max_usage_bytes{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",zone_name="hello"} 8 1395066363000
+# HELP container_memory_migrate Memory migrate status.
+# TYPE container_memory_migrate gauge
+container_memory_migrate{container_env_foo_env="prod",container_label_foo_label="bar",id="testcontainer",image="test",name="testcontaineralias",zone_name="hello"} 0 1395066363000
```

Comment:
I think that it will be good to modify fake value [here](https://github.com/google/cadvisor/blob/f17af505243c9f637b49a279f64de70a98edf3a3/metrics/prometheus_fake.go#L268) and set memory migrate to 1.